draft-ietf-rats-architecture.md

```diff
@@ -734,10 +734,29 @@ or interval when changes in operational status, health, and so forth occur.
 
 ## Endorsements
 
-An Endorsement is a secure statement that some entity (e.g., a manufacturer) vouches for the integrity of the
-device's signing capability.  For example, if the signing capability is in hardware, then
-an Endorsement might be a manufacturer certificate that signs a public key whose corresponding
-private key is only known inside the device's hardware.  Thus, when Evidence and such an Endorsement
+An Endorsement input to a Verifier is always required. An Endorsement is what allows a Verifier to trust an Attester and in turn to trust the Evidence produced by an Attester. Without Endorsements a Verifier cannot function.
+Said another way, an Attester for which there is no Endorsement is not useful since there is no good basis by which a Verifier can trust it.
+
+One of the Endorsements for an Attester will just about always contain cryptographic key material.
+This key material will correspond to key material that is put in the Attester.
+The key material is the basis for the Verifier trusting the Attester.
+This architecture does not dictate any particular cryptographic protocol for establishing this trust, hence the key material can be symmetric or asymmetric, a single key, multiple keys, key hierarchies, X.509 cert or chains or other.
+
+Endorsements are produced by an Endorser.
+The Endorser is often the manufacturer of the device that contains the Attester, but not necessarily so.
+For example, it could be an OS vendor that supplies Attester key material to the device/Attester manufacturer.
+
+An Endorsement must be conveyed to the Verifier in a secure way so that the Verifier does not accept fake or false Endorsements that would result in trusting Attesters and Evidence that should not be trusted.
```

Comment:
I still have the same objection to this text as before.

Reply:
Sorry, but I'm not able to figure out which previous comment applies to which line here.

Reply:
It was.  "This implies conveyance via a secure protocol, which I disagree is a must. As long as endorsements are signed, how they're conveyed is irrelevant... conveyance via an insecure protocol is fine."

Reply:
I've used the term "secure way" to attempt to cover all manner and means to secure the conveyance, not just to refer to channel-securing protocol like TLS and IPsec. I don't mean to say you must use TLS or IPsec. Sending a signed message over plain HTTP is fine and I would call that a "secure way". Hand carrying a USB flash drive from one company to another is also a "secure way".

So maybe we agree about the concept, but the words need improving?

Reply:
Maybe, text like this: if the Endorsement is not self-securing by signing or such, then some other secure means of conveyance should be used.

Reply:
Added a few sentences to clarify use of signed and unsigned Endorsements.